bokehjs/src/lib/testing.ts

```diff
@@ -0,0 +1,35 @@
+// Just a dumb key/value record for collecting arbitrary info for tests
```

Comment:
This this file has to be under `lib/`, when it's not part of the library?

Reply:
It is part of the library, the test suite calls this little function to be able to perform tests without a ton of boilerplate

Reply:
Then it's a part of testing code and shouldn't affect the library. Do you want every user pay a price for testing code? I presume it's a matter of convenience to put this code here, but it should be placed elsewhere (`bokeh._testing.util.selenium`).

Reply:
> Do you want every user pay a price for testing code? 

For 30 lines? Yes, I am absolutely, 100% ok with that.